src/wp-includes/class-wp-scripts.php

```diff
@@ -482,7 +498,11 @@ public function print_inline_script( $handle, $position = 'after', $display = tr
 		$output = trim( implode( "\n", $output ), "\n" );
 
 		if ( $display ) {
-			printf( "<script%s id='%s-js-%s'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), esc_attr( $position ), $output );
+			if ( 'after-non-standalone' === $position ) {
+				printf( "<script%s id='%s-js-after' type='text/template'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), esc_attr( $position ), $output );
```

Comment:
Since we're hard coding the position in the ID here, I think we also need to remove the `$position` variable that's being passed to the `printf` function.  We should probably add a unit test to cover this case as well. 

```suggestion
				printf( "<script%s id='%s-js-after' type='text/template'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), $output );
```

Reply:
Thanks @joemcgill. This is fixed now.